--- dev/examples/webenginewidgets/permissionbrowser/main.cpp.orig	2024-07-25 17:02:57.095976292 +0100
+++ dev/examples/webenginewidgets/permissionbrowser/main.cpp	2024-07-25 17:03:35.347433241 +0100
@@ -10,7 +10,7 @@
 {
     QCoreApplication::setOrganizationName("QtExamples");
     QApplication app(argc, argv);
-    app.setWindowIcon(QIcon(QString(u":AppLogoColor.png")));
+    app.setWindowIcon(QIcon(QString(":AppLogoColor.png")));
     MainWindow window(QUrl("qrc:/landing.html"));
     window.resize(1024, 768);
     window.show();
