diff --git a/qt6-qtwebengine.spec b/qt6-qtwebengine.spec
index 9fa700c..50ce713 100644
--- a/qt6-qtwebengine.spec
+++ b/qt6-qtwebengine.spec
@@ -51,7 +51,7 @@
 Summary: Qt6 - QtWebEngine components
 Name:    qt6-qtwebengine
 Version: 6.8.1
-Release: 2%{?dist}
+Release: 2a%{?dist}
 
 # See LICENSE.GPL LICENSE.LGPL LGPL_EXCEPTION.txt, for details
 # See also http://qt-project.org/doc/qt-5.0/qtdoc/licensing.html
@@ -92,6 +92,11 @@ Patch110: qtwebengine-webrtc-system-openh264.patch
 Patch111: qtwebengine-blink-system-openh264.patch
 Patch112: qtwebengine-media-system-openh264.patch
 
+## OT-SVG
+Patch1000: 0001-m122a-OT-SVG-hook.patch
+Patch1001: 0002-m122-use-FreeType-for-non-Linux-OT-SVG-support.patch
+Patch1002: qtwebengine-6.5.2-filename-too-long.patch
+
 # handled by qt6-srpm-macros, which defines %%qt6_qtwebengine_arches
 # FIXME use/update qt6_qtwebengine_arches
 # 32-bit arches not supported (https://bugreports.qt.io/browse/QTBUG-102143)
@@ -103,7 +108,7 @@ BuildRequires: qt6-srpm-macros
 BuildRequires: qt6-qtbase-devel
 BuildRequires: qt6-qtbase-private-devel
 # TODO: check of = is really needed or if >= would be good enough -- rex
-%{?_qt6:Requires: %{_qt6}%{?_isa} = %{_qt6_version}}
+%{?_qt6:Requires: %{_qt6}%{?_isa} >= %{_qt6_version}}
 BuildRequires: qt6-qtdeclarative-devel
 BuildRequires: qt6-qtlocation-devel
 BuildRequires: qt6-qtsensors-devel
@@ -319,7 +324,7 @@ Provides: bundled(v8) = 11.8.172.18
 # http://www.netlib.org/fdlibm/readme
 Provides: bundled(fdlibm) = 5.3
 
-%{?_qt6_version:Requires: qt6-qtbase%{?_isa} = %{_qt6_version}}
+%{?_qt6_version:Requires: qt6-qtbase%{?_isa} >= %{_qt6_version}}
 
 %description
 %{summary}.
@@ -369,6 +374,7 @@ Summary: Example files for qt6-qtpdf
 
 %prep
 %setup -q -n %{qt_module}-everywhere-src-%{version}%{?prerelease:-%{prerelease}} -a20
+rm %{SOURCE0}
 
 mv pulse src/3rdparty/chromium/
 
@@ -389,6 +395,11 @@ popd
 %patch -P111 -p1 -b .blink-system-openh264
 %patch -P112 -p1 -b .media-system-openh264
 
+## OT-SVG
+%patch -P1000 -p1 -b .OT-SVG
+%patch -P1001 -p1 -b .OT-SVG2
+%patch -P1002 -p1 -b .jumbo_build_long_path
+
 # delete all "toolprefix = " lines from build/toolchain/linux/BUILD.gn, as we
 # never cross-compile in native Fedora RPMs, fixes ARM and aarch64 FTBFS
 sed -i -e '/toolprefix = /d' -e 's/\${toolprefix}//g' \
@@ -429,7 +440,7 @@ cp -p src/3rdparty/chromium/LICENSE LICENSE.Chromium
 . /opt/rh/gcc-toolset-13/enable
 %endif
 export STRIP=strip
-export NINJAFLAGS="%{__ninja_common_opts}"
+export NINJAFLAGS="%{?_smp_mflags}"
 export NINJA_PATH=%{__ninja}
 
 %cmake_qt6 \
