diff --git a/qt6-qtwebengine.spec b/qt6-qtwebengine.spec
index 067fb61..c20c589 100644
--- a/qt6-qtwebengine.spec
+++ b/qt6-qtwebengine.spec
@@ -50,8 +50,8 @@
 
 Summary: Qt6 - QtWebEngine components
 Name:    qt6-qtwebengine
-Version: 6.7.1
-Release: 1%{?dist}
+Version: 6.8.0
+Release: 1a%{?dist}
 
 # See LICENSE.GPL LICENSE.LGPL LGPL_EXCEPTION.txt, for details
 # See also http://qt-project.org/doc/qt-5.0/qtdoc/licensing.html
@@ -90,6 +90,10 @@ Patch110: qtwebengine-webrtc-system-openh264.patch
 Patch111: qtwebengine-blink-system-openh264.patch
 Patch112: qtwebengine-media-system-openh264.patch
 
+## OT-SVG
+Patch1000: 0001-m122-OT-SVG-hook.patch
+Patch1001: 0002-m122-use-FreeType-for-non-Linux-OT-SVG-support.patch
+
 # handled by qt6-srpm-macros, which defines %%qt6_qtwebengine_arches
 # FIXME use/update qt6_qtwebengine_arches
 # 32-bit arches not supported (https://bugreports.qt.io/browse/QTBUG-102143)
@@ -101,7 +105,7 @@ BuildRequires: qt6-srpm-macros
 BuildRequires: qt6-qtbase-devel
 BuildRequires: qt6-qtbase-private-devel
 # TODO: check of = is really needed or if >= would be good enough -- rex
-%{?_qt6:Requires: %{_qt6}%{?_isa} = %{_qt6_version}}
+%{?_qt6:Requires: %{_qt6}%{?_isa} >= %{_qt6_version}}
 BuildRequires: qt6-qtdeclarative-devel
 BuildRequires: qt6-qtlocation-devel
 BuildRequires: qt6-qtsensors-devel
@@ -384,6 +388,10 @@ popd
 %patch -P111 -p1 -b .blink-system-openh264
 %patch -P112 -p1 -b .media-system-openh264
 
+## OT-SVG
+%patch -P1000 -p1 -b .OT-SVG
+%patch -P1001 -p1 -b .OT-SVG2
+
 # delete all "toolprefix = " lines from build/toolchain/linux/BUILD.gn, as we
 # never cross-compile in native Fedora RPMs, fixes ARM and aarch64 FTBFS
 sed -i -e '/toolprefix = /d' -e 's/\${toolprefix}//g' \
