diff --git a/qt6-qtwebengine.spec b/qt6-qtwebengine.spec
index 342f0cd..233620f 100644
--- a/qt6-qtwebengine.spec
+++ b/qt6-qtwebengine.spec
@@ -86,7 +86,7 @@
 Summary: Qt6 - QtWebEngine components
 Name:    qt6-qtwebengine
 Version: 6.9.2
-Release: 2%{?dist}
+Release: 2h%{?dist}
 
 # See LICENSE.GPL LICENSE.LGPL LGPL_EXCEPTION.txt, for details
 # See also http://qt-project.org/doc/qt-5.0/qtdoc/licensing.html
@@ -144,6 +144,11 @@ Patch201: qtwebengine-chromium-ppc64.patch
 # https://src.fedoraproject.org/rpms/chromium/c/c675db4ac0623d2d97344be0b3b2d9f1ac931446?branch=rawhide
 Patch202: chromium-130-size-assertions.patch
 
+## OT-SVG
+Patch1000: 0001-m122a-OT-SVG-hook.patch
+Patch1001: 0002-m130-use-FreeType-for-non-Linux-OT-SVG-support.patch
+Patch1002: qtwebengine-6.5.2-filename-too-long.patch
+
 # handled by qt6-srpm-macros, which defines %%qt6_qtwebengine_arches
 # FIXME use/update qt6_qtwebengine_arches
 # 32-bit arches not supported (https://bugreports.qt.io/browse/QTBUG-102143)
@@ -177,10 +182,10 @@ BuildRequires: git-core
 BuildRequires: qt6-srpm-macros
 BuildRequires: qt6-qtbase-devel
 BuildRequires: qt6-qtbase-private-devel
-%{?_qt6_version:Requires: qt6-qtbase%{?_isa} = %{_qt6_version}}
+%{?_qt6_version:Requires: qt6-qtbase%{?_isa} >= %{_qt6_version}}
 
 # TODO: check of = is really needed or if >= would be good enough -- rex
-%{?_qt6:Requires: %{_qt6}%{?_isa} = %{_qt6_version}}
+%{?_qt6:Requires: %{_qt6}%{?_isa} >= %{_qt6_version}}
 BuildRequires: qt6-qtdeclarative-devel
 BuildRequires: qt6-qtlocation-devel
 BuildRequires: qt6-qtsensors-devel
@@ -472,6 +477,7 @@ Requires: qt6-qtsvg%{?_isa}
 
 %prep
 %setup -q -n %{qt_module}-everywhere-src-%{qt_version}%{?prerelease:-%{prerelease}} -a20
+rm %{SOURCE0}
 
 mv pulse src/3rdparty/chromium/
 
@@ -498,6 +504,10 @@ pushd src/3rdparty/chromium
 %patch -P202 -p1
 popd
 
+## OT-SVG
+%patch -P1000 -F2 -p1 -b .OT-SVG
+%patch -P1001 -p1 -b .OT-SVG2
+%patch -P1002 -p1 -b .jumbo_build_long_path
 
 # delete all "toolprefix = " lines from build/toolchain/linux/BUILD.gn, as we
 # never cross-compile in native Fedora RPMs, fixes ARM and aarch64 FTBFS
@@ -549,7 +559,7 @@ src/3rdparty/chromium/build/linux/unbundle/replace_gn_files.py --system-librarie
 . /opt/rh/gcc-toolset-13/enable
 %endif
 export STRIP=strip
-export NINJAFLAGS="%{__ninja_common_opts}"
+export NINJAFLAGS="%{?_smp_mflags}"
 export NINJA_PATH=%{__ninja}
 
 # this follows the logic of the Configure summary to turn on and off
