diff --git a/qt6-qtwebengine.spec b/qt6-qtwebengine.spec
index 458d721..b94ba59 100644
--- a/qt6-qtwebengine.spec
+++ b/qt6-qtwebengine.spec
@@ -18,7 +18,7 @@
 %global use_system_py_six 1
 %endif
 
-%global use_system_re2 1
+%global use_system_re2 0
 
 # NEON support on ARM (detected at runtime) - disable this if you are hitting
 # FTBFS due to e.g. GCC bug https://bugzilla.redhat.com/show_bug.cgi?id=1282495
@@ -48,8 +48,8 @@
 
 Summary: Qt6 - QtWebEngine components
 Name:    qt6-qtwebengine
-Version: 6.6.1
-Release: 1%{?dist}
+Version: 6.7.0
+Release: 3f%{?dist}
 
 # See LICENSE.GPL LICENSE.LGPL LGPL_EXCEPTION.txt, for details
 # See also http://qt-project.org/doc/qt-5.0/qtdoc/licensing.html
@@ -59,7 +59,7 @@ URL:     http://www.qt.io
 # cleaned tarball with patent-encumbered codecs removed from the bundled FFmpeg
 # ./qtwebengine-release.sh
 # ./clean_qtwebengine.sh 6.4.1
-Source0: qtwebengine-everywhere-src-%{version}-clean.tar.xz
+Source0: qtwebengine-everywhere-src-%{version}-clean.tar.xz
 
 # cleanup scripts used above
 Source2: clean_qtwebengine.sh
@@ -94,6 +94,10 @@ Patch100: qtwebengine-webrtc-dlopen-h264.patch
 ## Upstreamable patches:
 Patch110: qtwebengine-blink-dlopen-h264.patch
 
+## OT-SVG
+Patch1000: 0001-m118-OT-SVG-hook.patch
+Patch1001: 0002-m118-use-FreeType-for-non-Linux.patch
+
 # handled by qt6-srpm-macros, which defines %%qt6_qtwebengine_arches
 # FIXME use/update qt6_qtwebengine_arches
 # 32-bit arches not supported (https://bugreports.qt.io/browse/QTBUG-102143)
@@ -102,7 +102,7 @@ BuildRequires: qt6-srpm-macros
 BuildRequires: qt6-qtbase-devel
 BuildRequires: qt6-qtbase-private-devel
 # TODO: check of = is really needed or if >= would be good enough -- rex
-%{?_qt6:Requires: %{_qt6}%{?_isa} = %{_qt6_version}}
+%{?_qt6:Requires: %{_qt6}%{?_isa} >= %{_qt6_version}}
 BuildRequires: qt6-qtdeclarative-devel
 BuildRequires: qt6-qtlocation-devel
 BuildRequires: qt6-qtsensors-devel
@@ -363,7 +367,7 @@ Summary: Example files for qt6-qtpdf
 %{summary}.
 
 %prep
-%setup -q -n %{qt_module}-everywhere-src-%{version}%{?prerelease:-%{prerelease}} -a20
+%setup -q -n %{qt_module}-everywhere-src-%{version}%{?prerelease:-%{prerelease}} -a20
 
 mv pulse src/3rdparty/chromium/
 
@@ -375,16 +379,20 @@ popd
 %patch3 -p1 -b .aarch64-new-stat
 %patch4 -p1 -b .qtwebengine-ffmpeg-first_dts
 
-%patch50 -p1 -b .fix-build.patch
+#%patch50 -p1 -b .fix-build.patch
 %if 0%{?fedora} && 0%{?fedora} >= 39
-%patch60 -p1 -b .fix-py-imp.patch
+#%patch60 -p1 -b .fix-py-imp.patch
 %endif
 
 ## upstream patches
-%patch100 -p1 -b .webrtc-dlopen-h264
+#%patch100 -p1 -b .webrtc-dlopen-h264
 
 ## upstreamable patches
-%patch110 -p1 -b .blink-dlopen-h264
+#%patch110 -p1 -b .blink-dlopen-h264
+
+## OT-SVG
+%patch1000 -p1 -b .OT-SVG
+%patch1001 -p1 -b .OT-SVG2
 
 # delete all "toolprefix = " lines from build/toolchain/linux/BUILD.gn, as we
 # never cross-compile in native Fedora RPMs, fixes ARM and aarch64 FTBFS
@@ -426,7 +434,7 @@ cp -p src/3rdparty/chromium/LICENSE LICENSE.Chromium
 . /opt/rh/gcc-toolset-12/enable
 %endif
 export STRIP=strip
-export NINJAFLAGS="%{__ninja_common_opts}"
+export NINJAFLAGS="%{?_smp_mflags}"
 export NINJA_PATH=%{__ninja}
 
 %cmake_qt6 \
