diff --git a/chromium/skia/BUILD.gn b/chromium/skia/BUILD.gn
index 4bdb828..6a8e1fa 100644
--- a/src/3rdparty/chromium/skia/BUILD.gn
+++ b/src/3rdparty/chromium/skia/BUILD.gn
@@ -597,6 +597,8 @@ component("skia") {
     sources += skia_sksg_sources
     sources += skia_shaper_primitive_sources
   }
+  import("//third_party/skia/modules/svg/svg.gni")
+  sources += skia_svg_sources
 }
 
 # Template for things that are logically part of :skia, but need to be split out
diff --git a/chromium/skia/ext/fontmgr_default_linux.cc b/chromium/skia/ext/fontmgr_default_linux.cc
index 76927e7..ed18f0f 100644
--- a/src/3rdparty/chromium/skia/ext/fontmgr_default_linux.cc
+++ b/src/3rdparty/chromium/skia/ext/fontmgr_default_linux.cc
@@ -7,10 +7,13 @@
 #include "third_party/skia/include/core/SkFontMgr.h"
 #include "third_party/skia/include/ports/SkFontConfigInterface.h"
 #include "third_party/skia/include/ports/SkFontMgr_FontConfigInterface.h"
+#include "third_party/skia/include/core/SkGraphics.h"
+#include "third_party/skia/modules/svg/include/SkSVGOpenTypeSVGDecoder.h"
 
 namespace skia {
 
 SK_API sk_sp<SkFontMgr> CreateDefaultSkFontMgr() {
+  SkGraphics::SetOpenTypeSVGDecoderFactory(SkSVGOpenTypeSVGDecoder::Make);
   sk_sp<SkFontConfigInterface> fci(SkFontConfigInterface::RefGlobal());
   return fci ? SkFontMgr_New_FCI(std::move(fci)) : nullptr;
 }
